Exercise/src/my_string.cpp

```diff
@@ -5,6 +5,10 @@
 #include "my_string.h"
 
 char* Exercise_1::CMyString::TryToAllocate(const size_t nLength) noexcept {
+	if(nLength == 0)
+	{
+		return nullptr;
+	}
 	char *pcAllocated;
```

Comment:
Why did you add noexcept?
If you try to handle exception in the function.
If you don't want to deal with exception just add noexcept after "new" and check ptr for nullptr after allocating

Reply:
Also imitialize this ptr with nullptr

Reply:
Fixed, by adding std::nothrow